tasks/reduce-snapshot/reduce-snapshot.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: reduce-snapshot
+  labels:
+    app.kubernetes.io/version: "0.0.1"
```

Comment:
nit pick but we usually go 0.1.0 to start

Reply:
updated